tests/components/test_auto_fill.py

```diff
@@ -18,9 +18,23 @@ def test_auto_fill_happy_path(kwik_e_mart_app, kwik_e_mart_app_path, qa_kwik_e_m
     assert_target_dialogue_state(convo, None)
 
 
+@pytest.mark.conversation
+def test_auto_fill_happy_path_validation(kwik_e_mart_app, kwik_e_mart_app_path, qa_kwik_e_mart):
+    """Tests a happy path for the app with gazetter validation."""
+    convo = Conversation(
+        app=kwik_e_mart_app, app_path=kwik_e_mart_app_path, force_sync=True
+    )
+    directives = convo.process("What's the store phone number?").directives
+    assert_target_dialogue_state(convo, "send_store_phone")
+    assert_reply(directives, "Which store would you like to know about?")
+
+    directives = convo.process("the store on elm street").directives
```

Comment:
do we need to make assertions for these directives? I am not sure what we are testing here